Duality/Resources/Font.cs

```diff
@@ -382,12 +387,15 @@ public void UpdateKerningData()
 
 				for (int i = 0; i < this.glyphs.Length; ++i)
 				{
-					PixelData glyphTemp = this.GetGlyphBitmap(this.glyphs[i].Glyph);
+					PixelData glyphTemp_ = this.GetGlyphBitmap(this.glyphs[i].Glyph);
+					PixelData glyphTemp = new PixelData(this.glyphs[i].Width, this.glyphs[i].Height + this.glyphs[i].OffsetY);
+					glyphTemp_.DrawOnto(glyphTemp, BlendMode.Solid, 0, this.glyphs[i].OffsetY);

```

Comment:
Creating another temporary glyph bitmap to draw it with an offset is really slow. For performance reasons, it would be much better to account for the Y offset math-wise in the following algorithm.